### DISCUSSION TITLE
Add granular invite links to team screen

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#6190] [Ideas]
**Background**
Our current team invite screen doesn't show invite links, and doesn't allow opening a team to anyone with a link.

Current:
![image](https://user-images.githubusercontent.com/9154902/154373936-e122c5eb-a043-49b4-b40b-2b5a7d407a59.png)

Proposed (this is just a sketch, not a pixel perfect design):
![image](https://user-images.githubusercontent.com/9154902/154372780-1cc4940b-5a98-46d0-933d-853a6a837ec6.png)

**Details:**
With this change, we'll be able to address each of these use cases:
1. Don't invite anyone
2. Invite only who I add via email
3. Allow anyone from my domain to join
4. Easily grab developer invite URL (new use case)
5. Easily grab user invite URL (new use case)
6. Allow anyone with the invite link to join (new use case)

**Comment by jonbell-lot23** [comment]
Looks like this is closed

**Comment by ceceliacreates** [comment]
Hey @jonbell-lot23 this was closed but it doesn't look like the PRs have landed. Should it stay open until it's implemented or did we decide not to move forward with it?

**Comment by jonbell-lot23** [comment]
Not sure! Re-opened

**Comment by jonbell-lot23** [comment]
Removing myself from this ticket, nothing for me to do

**Comment by jonbell-lot23** [comment]
Adding to the sprint for visibility
